assets/_scss/components/_disclaimer.scss

```diff
@@ -1,16 +1,33 @@
 .usa-disclaimer {
+  @include media($small-screen) {
+    font-size: $h5-font-size;
+  }
   background-color: $color-gray-lighter;
-  font-size: 1.5rem;
+  font-size: $h6-font-size;
   padding: {
     bottom: .5rem;
     top: .5rem;
   }
 }
 
+// This hides the stage note of the disclaimer in anything smaller than 690px
 .usa-disclaimer-stage {
-  @include media($medium-screen) {
+  @include media(690px) {
```

Comment:
Ok switched to large breakpoint in: 63c557a. I think it's better if it disappears early than the bad experience of it breaking to two lines. People can always adjust this breakpoint to something that works better for themselves.